test/acceptance/integration/sth/notifications.feature

```diff
@@ -0,0 +1,241 @@
+
+  @happy_path
+  Scenario: stored new notifications in mongo from context broker with metadata
+    Given copy properties.json file from "epg_properties.json" and sudo local "false"
+  # The next line is commented because at the moment the rpm package does not exists. STH is executed manually
+  #  And restart sth service. This execution is only once "true"
+    And verify if sth is installed correctly
+    And mongo is installed correctly
+    And service "test_frb", service path "/test", entity type "room", entity_id "room2", with attribute number "1", attribute name "random" and attribute type "celcius"
+    When receives a notification with attributes value "random number=2", metadata value "True" and content "json"
+    Then validate that the attribute value and type are stored in mongo
+    And validate that the aggregated value is generate by resolution "month" in mongo
+    #And delete database in mongo
+
+  @resolutions
+  Scenario Outline: stored new notifications in mongo from context broker with metadata with several resolutions
+    Given copy properties.json file from "epg_properties.json" and sudo local "false"
+  # The next line is commented because at the moment the rpm package does not exists. STH is executed manually
+  #  And restart sth service. This execution is only once "true"
+    And verify if sth is installed correctly
+    And mongo is installed correctly
+    And service "test_resolutions", service path "/test", entity type "room", entity_id "room2", with attribute number "1", attribute name "random" and attribute type "celcius"
+    When receives a notification with attributes value "random number=2", metadata value "True" and content "json"
+    Then validate that the attribute value and type are stored in mongo
+    And validate that the aggregated value is generate by resolution "<resolution>" in mongo
+    And delete database in mongo
+  Examples:
+    | resolution |
+    | month      |
+    | day        |
+    | hour       |
+    | minute     |
+    | second     |
+
+  @service
+  Scenario Outline:  stored new notifications in mongo from context broker with metadata with several services
+    Given copy properties.json file from "epg_properties.json" and sudo local "false"
+  # The next line is commented because at the moment the rpm package does not exists. STH is executed manually
+  #  And restart sth service. This execution is only once "true"
+    And verify if sth is installed correctly
+    And mongo is installed correctly
+    And service "<service>", service path "/test", entity type "room", entity_id "room2", with attribute number "1", attribute name "random" and attribute type "celcius"
+    When receives a notification with attributes value "random number=2", metadata value "True" and content "json"
+    Then validate that the attribute value and type are stored in mongo
+    And validate that the aggregated value is generate by resolution "month" in mongo
+    And delete database in mongo
```

Comment:
The same of the comment before, consider clean the database before or after the test, in the terrain